photon-ml/src/main/scala/com/linkedin/photon/ml/data/GameConverters.scala

```diff
@@ -63,7 +63,7 @@ object GameConverters {
     // Add unique id
     val recordsWithUniqueId = data.withColumn(UNIQUE_ID_COLUMN_NAME, monotonicallyIncreasingId)
 
-    recordsWithUniqueId.map { row: Row =>
+    recordsWithUniqueId.rdd.map { row: Row =>
```

Comment:
Good to know!

Please feel free to merge this PR, as I no longer have the write access to merge PR for Photon.